### ISSUE TITLE
Made small updates, might want to use it

### ISSUE THREAD

**Issue by Skquark** [RatelHub/rflutter_alert#2]
Great alert dialog, just what I was hoping someone would make since I was bored with the standard material dialog that I just couldn't customize enough.  I hacked a few changes that I needed for my usage, and it's simple enough to offer in the next version.. I needed a general widget content option to provide my own form fields in the popup for small entry like a dropdown or input text.  Really easy need, so I added this to my Alert Class:
`final Widget content;`
then in _buildDialog, right after the desc text, added this:
`content == null ? Container() : content,`

I also noticed there was an image parameter to use instead of the premade images for the AlertType, but it wasn't being used right.  In _getImage() you assigned Widget response = image; but then response was overridden based on the AlertType specified.  I wanted there to be something like AlertType.custom but rather than add that myself, I just modified this part of the switch statement:
`case AlertType.none:`
`        response = image == null ? Container() : image;`
`        break;`
Very easy fix, it now looks the way I wanted it and I'm quite happy with the results so I just wanted to share what I changed, adapt it as you see fit.  Thanks.

**Comment by ibrahimdevs** [comment]
@Skquark Thank you for the great contribution and fix. We will add the custom content feature, fix the image issue . And easier way of creating Alert Buttons will available next release ( V1.0.2 ) in a few days.

We are always welcome to new ideas and even bugs 👍

**Comment by Skquark** [comment]
Good stuff, thanks.  I was fine-tuning the look of mine a little more, and ran into another missing parameter that needed to be exposed.  Another easy one, changing the background color and elevation.  Here's my simple adds to AlertStyle:
`  final Color backgroundColor;`
`  final double elevation;`
then in _buildDialog AlertDialog(
`      backgroundColor: style.backgroundColor,`
`      elevation: style.elevation,`
Now it looks better to my eyes, wanted to be off-white and with more shadow... worth it.  Thanks again.
I was also thinking it would be nice to have a dialogPadding separate from buttonAreaPadding to tweak it just right..

**Comment by ibrahimdevs** [comment]
@Skquark Have you tried your backgroundColor and elevation changes? If you tried, can you share your code? Because we are not sure what is the best practice of changing the background color. And we didn't set any shadow, so we don't know how to change shadow?
Thanks,

**Comment by Skquark** [comment]
No problem.. It's part of the AlertDialog that you're already using, so was very convenient, waiting to be used.  Here's where I put that code:
```
Widget _buildDialog() {
    return AlertDialog(
      shape: style.alertBorder ?? _defaultShape(),
      backgroundColor: style.backgroundColor,
      elevation: style.elevation,
```
That simple.  It's alright when it's null too, just goes to the default when it is.
Just got your latest update.... Thumbs up.  Thanks for your work, great addition to the Flutter ui..

**Comment by Skquark** [comment]
I've got another minor suggestion if you don't mind me offering.. I was building an Alert with AlertType.none and I expected to see the title all the way on the top, but there was a lot of padding there, just didn't look right.  Tracked it down easily, here's what I changed in _buildDialog:
```
                    _getImage(),
                    SizedBox(
                      height: type == AlertType.none ? 0 : 15,
                    ),
```
There was a lot of top padding from the Close button too, but I just set isCloseButton: false and it was good enough.  Personally, I'd recommend putting the CloseButton inside a Stack then Positioned with a top right, that's how I've done it in a Card with a remove X and it worked nicely.  I could write up that mod for you real quick if you'd like, but there'd probably be other paddings that need adjustment after that, and it's not crucial.  Anyways, having fun with this component, I'll let you know if there's any other tweaks as I build a few more things with it.  Thanks.
